### PR TITLE
add `remarks` for review enhancement

### PR DESCRIPTION
**Pull request by hixi**
Unfortunately, I had to make the changes on top of e35e4596523673f6856dd2b509b359a75fabba02 (1.2.3 Release), since the current master stopps work with `index out of range` on line https://github.com/mapsme/osm_conflate/blob/af20b5e506d53bf20b9181ec8f544cfad2bd65da/conflate/conflate.py#L706.

Feel free to edit or/and update this PR as you please. Questions and remarks welcome.

### CODE REVIEW

File: conflate/conflate.py [hunk 1]
```diff
@@ -78,11 +79,13 @@ def __init__(self, ptype, pid, version, lat, lon, tags=None, categories=None):
         self.members = None
         self.action = None
         self.categories = categories or set()
+        self.remarks = None
 
     def copy(self):
         """Returns a copy of this object, except for members field."""
         c = OSMPoint(self.osm_type, self.osm_id, self.version, self.lat, self.lon, self.tags.copy())
         c.action = self.action
+        c.remarks = self.remarks
```
Comment (Zverik) [inline conflate/conflate.py:88]: Looks like I forgot to copy the `categories` set. Could you do that please?

File: conflate/conflate.py [hunk 2]
```diff
@@ -625,6 +628,8 @@ def format_change(before, after, ref):
                     else:
                         props['tags_changed.{}'.format(k)] = '{} -> {}'.format(v0, v1)
             props['marker-color'] = MARKER_COLORS[marker_action]
+            if ref and ref.remarks:
+                props['__remarks__'] = ref.remarks
```
Comment (Zverik) [inline conflate/conflate.py]: I think `__` is excessive: just use `remarks` please.

**Review by Zverik — CHANGES_REQUESTED** [review]

File: conflate/conflate.py (hunk 2 above)
Comment (hixi) [reply conflate/conflate.py]: A quick search on overpass tells me, that there are (at least) three nodes in OSM which possess a tag named "remarks" (and a few more with "remark"). I think in order to ensure collisions, we should stick to a highly unlikely name. I agree, that using a `__` is a bit on the overly cautious side, but using plain `remarks` seems too lax to me. What about `_remarks`, seems a good compromise between "clash-freeness" and not too long?

Comment (Zverik) [reply conflate/conflate.py]: Tags are prefixed with `tags.`, `tags_changed.` etc.

Comment (hixi) [reply conflate/conflate.py]: True, forgot about that. Thank you for the reminder, I'll change it accordingly!

File: conflate/conflate.py (hunk 1 above)
Comment (hixi) [reply conflate/conflate.py:88]: Of course. Done.

Comment (Zverik) [reply conflate/conflate.py:88]: Thanks, though you need to use `self.categories.copy()`, otherwise you're just copying a pointer to the same set.

Comment (hixi) [reply conflate/conflate.py:88]: My bad, corrected. Thanks for your time and the effort on this project!

**Comment by Zverik** [comment]
Thanks!

**Pull request merged**